widget/entry.go

```diff
@@ -474,6 +474,19 @@ func (e *Entry) SetText(text string) {
 	e.updateCursorAndSelection()
 }
 
+// Appends the text to the end of the entry
```

Comment:
Sorry, but the comment should begin "Append" for the linter to be happy - i.e. "Append will add the given text to the end of the Entry"...